cacheops/query.py

```diff
@@ -32,6 +31,13 @@
 from .transaction import transaction_states
 from .signals import cache_read
 
+if settings.CACHEOPS_PICKLE_LIB == 'pickle':
+    import pickle
+elif settings.CACHEOPS_PICKLE_LIB == 'dill':
+    import dill as pickle
+else:
+    raise ValueError('settings.CACHEOPS_PICKLE_LIB invalid value (use pickle or dill)')
```

Comment:
Use `import_string()` and do it once somewhere, maybe in `conf`. Using those at module load time will make it harder to test.

See how `CACHEOPS_PREFIX` is handled.